markets/bfp-market/cannonfile.toml

```diff
@@ -155,6 +149,18 @@ factory.PerpAccountProxy.arg = 2
 
 # --- Mocks --- #
 
+[contract.SettlementHookMock]
+artifact = "contracts/mocks/SettlementHookMock.sol:SettlementHookMock"
+args = ["<%= contracts.InitialPerpMarketProxy.address %>"]
+depends = ["contract.InitialPerpMarketProxy"]
+salt = "1"
+
+[contract.SettlementHook2Mock]
+artifact = "contracts/mocks/SettlementHookMock.sol:SettlementHookMock"
+args = ["<%= contracts.InitialPerpMarketProxy.address %>"]
+depends = ["contract.InitialPerpMarketProxy"]
```

Comment:
As `contracts.InitialPerpMarketProxy` is already in args auto-dependency resolution works fine. No need for manual depends
```suggestion
```

Reply:
PS: here and in other places, not just in this line. Reckon all of the `depends` can go.